<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewIndexDialog</class>
 <widget class="QDialog" name="NewIndexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a New Image Index</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Feature type: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="featureTypeCombo">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Color Histogram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Color Moments</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fuzzy Histogram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fuzzy Spatial Histogram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lu et al. v2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CEDD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Zhang et al.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HMMD quantization from CSD</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="blankFeatureOptions">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Feature options: </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="colorHistogramOptions_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="colorHistogramOptions">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Color histogram options:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Color model:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="colorModelCombo">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>RGB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>YUV</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HSV</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HSL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>YIQ</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>XYZ</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LAB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LUV</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HY</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IHLS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HMMD</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Color quantization:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cq1spin">
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cq2spin">
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cq3spin">
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cq4spin">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Histogram type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="histogramTypeCombo">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Combined (Linked)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Split</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Histogram normalization:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="normalizationCombo">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Max</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Both</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Histogram quantization:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="histogramQuantizationSpin">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>64</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>bits per bin</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="cumulativeCheck">
          <property name="text">
           <string>Histogram cumulative</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Distance metric:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="distanceMetricCombo">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Indexing structure:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="indexerCombo">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Sequential indexer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ANN indexer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LSH indexer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tree indexer</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewIndexDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewIndexDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
